README.md

```diff
@@ -64,4 +71,13 @@ protocol, host and port.
 
 `client_certificates` is a list of client certificates to register to LXD. When a certificate
 has been processed by the `provides` side, it should be available in the
-`registered_certificates` list on the relation.
\ No newline at end of file
+`registered_certificates` list on the relation.
+
+## Security
+Security issues in the operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File) on the [Anbox Cloud](https://bugs.launchpad.net/anbox-cloud) project. Please do not file GitHub issues about security issues.
+
+## Contributing
+Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/nats-operator/blob/main/CONTRIBUTING.md) for developer guidance.
```

Comment:
This is linking to the CONTRIBUTING.md of the nats-operator. We need to have our own here and link it.